scripts/benchmark_stats.py

```diff
@@ -39,6 +41,8 @@ def __str__(self):
                 f'{formatted_percentiles}<br />'
                 f'GETs (99th): {self.average_get}ms<br />'
                 f'POSTs (99th): {self.average_post}ms<br /><br />'
+                f'PDF: {str(self.pdf_percentile)+"ms" if self.pdf_percentile else "N/A"}<br />'
```

Comment:
Simpler way to do this would be to make pdf_percentile return string with `ms` already attached since you do this conversion everywhere anyway (line 44, 58)?  so then it would be as simple as: `PDF: {self.pdf_percentile or "N/A"}<br />` so it choose "N/A" when `self.pdf_percentile` has falsey value.

Reply:
Updated to use a new function to format the string 👍

Reply:
This is useful but can it be extended to just be format_percentiles so it handles the adding of `ms` for all of them? Would make it more consistent then

Reply:
Took a slightly different approach that just updating the method, happy to change to an alternative if you had another solution in mind 👍